ppb_vector/vector2.py

```diff
@@ -46,6 +46,10 @@ def __rmul__(self, other):
         if isinstance(other, Number):
             return Vector2(self.x * other, self.y * other)
 
+    def __xor__(self, other):
+        assert(isinstance(other, Vector2))
```

Comment:
Please don't do this. Assertion errors in running code aren't meaningful for the end user.

Reply:
This should be redone with my PR.